install/playbooks/roles/external-ip/tasks/main.yml

```diff
@@ -44,19 +35,19 @@
 
 - name: Run the script to get the external IP address automatically
   tags: facts
-  when: network.external_ip == "auto"
+  when: network.auto_detect_ip
```

Comment:
This test is also used in `common/roles/external-ip-type/tasks/main.yml`, see: https://github.com/progmaticltd/homebox/pull/199/commits/490db12213735d395fc4b44d65d8c539068dfa3d (in #199).